src/ui/Quickview/QuickviewDocument.ts

```diff
@@ -200,6 +200,12 @@ export class QuickviewDocument extends Component {
   private writeToIFrame(iframe: HTMLIFrameElement, content: HTMLDocument);
   private writeToIFrame(iframe: HTMLIFrameElement, content: String);
   private writeToIFrame(iframe: HTMLIFrameElement, content: any) {
+
+    if (content.type == HTMLDocument) {
+      _.each($$(content.body).findAll('a'), (link) => {
+        link.setAttribute('target', '_top');
```

Comment:
I'm reading the docs on this attribute and it appears that it will usually open the link in the current context. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a Didn't we talked about making it open in a new window always?